### PR TITLE
Replace comment with real method signature

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]

File: src/Persistence/ObjectManager.php [hunk 1]
```diff
@@ -122,6 +118,9 @@ public function getMetadataFactory(): ClassMetadataFactory;
      */
     public function initializeObject(object $obj): void;
 
+    /** Helper method to check whether a lazy loading proxy or persistent collection has been initialized. */
+    public function isUninitializedObject(mixed $value): bool;
```
Comment (derrabus) [inline src/Persistence/ObjectManager.php:122]: Is this really `mixed` or should this be an object at least?

Comment (greg0ire) [reply src/Persistence/ObjectManager.php:122]: Really `mixed`, see discussion on slack channel proxy-integration

Comment (greg0ire) [reply src/Persistence/ObjectManager.php:122]: back at my desktop, here is the link: https://doctrine.slack.com/archives/C045LBBNALQ/p1716360854270869

Comment (derrabus) [reply src/Persistence/ObjectManager.php:122]: 😕 

Not happy about that, but if that's the consensus, … 🤷‍♂️

**Pull request merged**